From 3efc4ec8d1cfe44213adfc1d852a1e8fd2b7831c Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Wed, 26 Sep 2012 15:28:21 +0800
Subject: [PATCH 065/602] ENGR00215174 disable vpu usage now for stagefright.

vpu codecs has not been added to stagefright.
disable it before added.

Change-Id: If4cd96d1060f15a8a65bbd58a70ae46356018646
Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 imx6/soc/imx6dq.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/soc/imx6dq.mk b/imx6/soc/imx6dq.mk
index adfb801..37523d1 100644
--- a/imx6/soc/imx6dq.mk
+++ b/imx6/soc/imx6dq.mk
@@ -3,7 +3,7 @@
 #
 
 BOARD_SOC_TYPE := IMX6DQ
-BOARD_HAVE_VPU := true
+BOARD_HAVE_VPU := false
 HAVE_FSL_IMX_GPU2D := true
 HAVE_FSL_IMX_GPU3D := true
 HAVE_FSL_IMX_IPU := true
-- 
1.8.0

